docs/conf.py

```diff
@@ -6,6 +6,7 @@
 #pylint: skip-file
 
 sys.path.insert(0, os.path.abspath('..'))
+from lore.settings import VERSION
```

Comment:
Something like it, but this time rtd is happy with the result.  See http://lore.readthedocs.org/en/feature-pw-add_release_process_notes/